drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -289,13 +379,26 @@ static int lis2dw12_init(const struct device *dev)
 		return ret;
 	}
 
+#ifdef CONFIG_LIS2DW12_BW_FILT
+    LOG_INF("Setting filter bandwidth: %d", (int)cfg->bw_filt);
+    lis2dw12_filter_bandwidth_set(ctx, cfg->bw_filt);
```

Comment:
Wherte is bw_filt assigned? Should be taken from DT right?

Reply:
This part was missing, indeed. I have pushed a new commit where this is fixed.

Reply:
OK, and I think you can get rid of CONFIG_LIS2DW12_BW_FILT in Kconfig.
Again, the default will be 0 if not defined in a particular driver instance.